content/docs/next-release/spm.md

```diff
@@ -69,12 +69,25 @@ as such, is only relevant to the Jaeger Query component (and All In One).
 
 {{<mermaid align="center">}}
 graph
+    OTLP_EXPORTER[OTLP Exporter] --> TRACE_RECEIVER
+
+    subgraph Service
+        subgraph OpenTelemetry SDK
+            OTLP_EXPORTER
+        end
+    end
+
     TRACE_RECEIVER[Trace Receiver] --> |spans| SPANMETRICS_PROC[Spanmetrics Processor]
     TRACE_RECEIVER --> |spans| TRACE_EXPORTER[Trace Exporter]
+    TRACE_EXPORTER --> |spans| COLLECTOR[Jaeger Collector]
     SPANMETRICS_PROC --> |metrics| PROMETHEUS_EXPORTER[Prometheus/PromethesusRemoteWrite Exporter]
-    UI[Jaeger UI] --> QUERY
-    QUERY[Jaeger Query Service] --> METRICS_STORE[Metrics Storage]
-    PROMETHEUS_EXPORTER --> |metrics| METRICS_STORE
+    PROMETHEUS_EXPORTER --> |metrics| METRICS_STORE[(Metrics Storage)]
+
+    COLLECTOR --> |spans| SPAN_STORE[(Span Storage)]
+    SPAN_STORE --> QUERY[Jaeger Query]
```

Comment:
Reversing the arrow directions to represent data flow rather than event initiation achieves two goals:
- Avoid confusion between what Jaeger Collector and Query do. It could lead users to think that both components write data to trace storage.
- Prevents arrow/edge crossover which will result if jaeger query -> span storage.